pom.xml

```diff
@@ -42,6 +42,7 @@
     <beam-vendor-guava.version>0.1</beam-vendor-guava.version>
     <hamcrest.version>1.3</hamcrest.version>
     <java.version>1.8</java.version>
+    <jib.maven.version>1.1.2</jib.maven.version>
```

Comment:
This pom.xml should have zero changes right?

Reply:
Correct, proposed changes have been removed.